src/features/auth/forgottenPassword/ResetPasswordExpiredLink.tsx

```diff
@@ -22,25 +23,33 @@ import { contactSupport } from '../support.services'
 type Props = StackScreenProps<RootStackParamList, 'ResetPasswordExpiredLink'>
 
 export function ResetPasswordExpiredLink(props: Props) {
+  const { email } = props.route.params
   const { navigate } = useNavigation<UseNavigationType>()
-
+  const { refetch, isFetching } = useQuery(
+    'resetPasswordExpiredLink',
+    resetPasswordExpiredLinkQuery,
+    {
+      cacheTime: 0,
+      enabled: false,
+      onSuccess: () => {
+        navigate('ResetPasswordEmailSent', { email })
+      },
+    }
+  )
   function goToHomeWithoutModal() {
     navigate('Home', NavigateToHomeWithoutModalOptions)
   }
-
-  async function resendEmailForResetPassword() {
-    const { email } = props.route.params
-    analytics.logResendEmailResetPasswordExpiredLink()
-    await api
-      .postnativev1requestPasswordReset({ email })
-      .then(() => {
-        navigate('ResetPasswordEmailSent', { email })
-      })
-      .catch((error) => {
-        Alert.alert(error.message)
-      })
+  async function resetPasswordExpiredLinkQuery() {
+    try {
+      analytics.logResendEmailResetPasswordExpiredLink()
+      await api.postnativev1requestPasswordReset({ email })
+    } catch (err) {
```

Comment:
Nitpick : tu peux prepender la variable par "_" pour montrer qu'elle est inutilisée (et que c'est volontaire)
```suggestion
    } catch (_err) {
```